commands/bootstrap/inputs.go

```diff
@@ -596,7 +613,47 @@ func initializeInputs(inputFilePath, resourcesPath string, configuration config.
 		}
 	}
 
+	if inputsPath == "" {
+
+		bSlice, err := yaml.Marshal(inputValues)
+
+		inputsPathOut := deploymentName + ".yaml"
+
+		count := 1
+		for {
+			if _, err := os.Stat(inputsPathOut); os.IsNotExist(err) {
+				break
+			}
+			inputsPathOut = deploymentName + "_" + strconv.Itoa(count) + ".yaml"
+			count++
+		}
+
+		if _, err := os.Stat("inputsaves/"); os.IsNotExist(err) {
+			err = os.Mkdir("inputsaves/", 0755)
+			if err != nil {
+				log.Fatal("cannot create inputsave directory.", err)
+			}
+		}
+
+		file, err := os.Create("inputsaves/" + inputsPathOut)
```

Comment:
We may use os.Open() to manage permissions and create the file in 0700 mode to not provide read access to group or others as the file contains sensitive information like private key content.

Reply:
Done, to perm 0600 mode since we do not need execution on a config file, and to 0700 for the inputsaves/ directory